services/contract_update.proto

```diff
@@ -109,4 +109,26 @@ message ContractUpdateTransactionBody {
      * account. Otherwise it updates the contract's auto-renew account to the referenced account.
      */
     AccountID auto_renew_account_id = 12;
+
+    /**
+     * ID of the new account or node to which this contract is staking. It can never happen that staked_account_id
+     * is non-null and staked_node_id is non-negative at the same time.
+     */
+    oneof staked_id {
+
+        /**
+         * ID of the new account to which this contract is staking. The default value is null.
```

Comment:
I don't think the oneof by itself provides a way to clear a value. So we still need to call `contractUpdateBuilder.setStakedAccountId(AccountID.getDefaultInstance())` to indicate both that you want to changed the staked ID and that it should be cleared.

```suggestion
         * ID of the new account to which this contract is staking. If set to the sentinel <tt>0.0.0</tt> AccountID, this field removes the contract's staked account.
```